suites/pacific/rgw/tier-3_rgw_multisite_archive_with_haproxy.yaml

```diff
@@ -457,3 +457,33 @@ tests:
       name: test LC from primary to secondary
       polarion-id: CEPH-10737
       comments: bug-1991808, test LC rules from primary to secondary
+  - test:
+      clusters:
+        ceph-pri:
+          config:
+            set-env: true
+            script-name: test_bucket_lifecycle_object_expiration_transition.py
+            config-file-name: test_lc_transition_with_prefix_rule_haproxy.yaml
+            run-on-haproxy: true
+            timeout: 300
+      desc: test LC transition on multisite
+      module: sanity_rgw_multisite.py
+      name: test LC transition on multisite
+      polarion-id: CEPH-10737
+      comments: bug-1991808, test LC rules transition from primary to secondary
+
+  - test:
+      clusters:
+        ceph-pri:
+          config:
+            set-env: true
```

Comment:
I think, once set-env has been set no need to set it again in next testcase
since both running in same cluster

Reply:
addressed.